.eslintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   parser: '@typescript-eslint/parser', // Specifies the ESLint parser
   plugins: [
     'cucumber',
-    '@typescript-eslint'
+    '@typescript-eslint',
```

Comment:
Why a trailing comma here, and then not down below? Maybe we should remove trailing commas altogether

Reply:
Okay, I'll remove all the trailing commas for consistency

Reply:
But actually more lines already have trailing commas. What to do ? @lewisdaly